Sources/CNIOSHA1/c_nio_sha1.c

```diff
@@ -44,15 +42,15 @@
  * implemented by Jun-ichiro itojun Itoh <itojun@itojun.org>
  */
 
-#include <sys/cdefs.h>
-
+#include "include/CNIOSHA1.h"
+#include <string.h>
+#define bzero(b,l) memset((b), '\0', (l))
```

Comment:
These redefines emit warnings on macOS due to redefining the macro. I think we'll need a different solution.

Reply:
@compnerd I assume these warnings are still appearing.

Reply:
Yup, they are. I'd like us to address this before we merge.

Reply:
I completely missed this bit.  Um, Im not sure what a good way to handle that is.  Personally, I think that we should just silence the warning - macro redefinition is portable - it is part of the standard as of C99.  We could also simply just set the C standard to C99 rather than ignoring the warning.  Thoughts?

Reply:
I'm fine with setting the C standard to C99, I cannot imagine it would break anything.